paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/LookupJoinITCase.java

```diff
@@ -517,6 +517,29 @@ public void testRetryLookup() throws Exception {
         iterator.close();
     }
 
+    @Test
+    public void testAsyncRetryLookup() throws Exception {
```

Comment:
It seems this unit test does not have difference with existed `testRetryLookup`, do you think we should test the `async` behavior with existing sync lookup tests?

Reply:
It is better to separate them. Every test should launch a streaming job. It is good to just launch one job in one test.